index.php

```diff
@@ -146,6 +146,19 @@
 							'classes' => [ 'form-repos-field' ],
 						]
 					),
+					new OOUI\FieldLayout(
+						new OOUI\CheckboxInputWidget( [
+							'name' => 'proxy',
+							'value' => 1,
+							'selected' => false
```

Comment:
Should probably be disabled by default.

Reply:
Disabled or checked (selected)?

Reply:
Sorry, unchecked (selected=false)

Reply:
Sorry.. still a little confused. what do you want me to change here? You want me to make it selected = true ?